src/listr/tasks/registerMasternodeTaskFactory.js

```diff
@@ -67,11 +60,19 @@ function registerMasternodeTaskFactory(
           // eslint-disable-next-line no-param-reassign
           task.output = `${ctx.fundingPrivateKeyString} imported.`;
         },
+        options: { persistentOutput: true },
       },
       {
         title: 'Sync Core with network',
         enabled: () => config.get('network') !== NETWORK_LOCAL,
-        task: async (ctx) => waitForCoreSync(ctx.coreService),
+        task: async (ctx, task) => {
+          function updateSyncStatus(verificationProgress) {
+            // eslint-disable-next-line no-param-reassign
+            task.output = `${(verificationProgress * 100).toFixed(2)}% complete`;
+          }
+          await waitForCoreSync(ctx.coreService, updateSyncStatus);
```

Comment:
There is no need to pass a named function. You can pass just a closure (array function).

Reply:
OK, thanks for the above explanations. Now I was finally able to match the pattern used in the other functions. But I still can't understand _why_ it works...